assets/sass/modules/_consent.scss

```diff
@@ -0,0 +1,141 @@
+.consent-beacon {
+
+  .logo-wrapper {
+    width: 71px;
+    height: 71px;
+    position: absolute;
+    border-radius: 50%;
+    top: 7px;
+    bottom: 0;
+    background-color: #fff;
```

Comment:
shall you use color variable from courage `courage/assets/sass/helpers/_variables.scss`?
though the design is not final, we shall follow color pattern?

Reply:
oh this is the new design (see video), will update with variables from _.variables.scss, good point